lib/generators/templates/spec.erb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+RSpec.describe <%= class_name %> do
```

Comment:
One more minor thing: I would change this line to include `type: :interactor`.

No real drawbacks, near as I can see.  It doesn't hurt anyone who doesn't use the interactor type, and it would be helpful to anyone who does.